common/darwin/Classes/FlutterRTCMediaStream.m

```diff
@@ -515,7 +515,9 @@ -(void)mediaStreamTrackRelease:(RTCMediaStream *)mediaStream  track:(RTCMediaStr
 
 -(void)mediaStreamTrackSetEnabled:(RTCMediaStreamTrack *)track : (BOOL)enabled
 {
-  if (track && track.isEnabled != enabled) {
+    // fix for issue
```

Comment:
Sorry, but as for me, this is a meaningless comment. Could you please clarify in the comment for what exactly issue?